### ISSUE TITLE
come up with new, concise names for methods or at least make them consistent

### ISSUE THREAD

**Issue by parksw3** [parksw3/dynamicaltruncation#58]

**Comment by athowes** [comment]
On this, I might prefer the function in `epidist` to be called something shorter than `latent_...` (and so on) if it's possible.

**Comment by seabbs** [comment]
Current_best_idea_but_we_arent_happy_about_it.

I think we also have a different name in epidist especially if we make a package wide assumption all models account for censoring and truncation?

**Comment by athowes** [comment]
Yes agree that when you don't have all the other models it's easier to call it by something simpler. Sometimes things even have names that are almost nonsensical. `mgcv` -- what's a computation vehicle?
